src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -2808,4 +2808,25 @@ Zero-width positive lookbehind assertions are typically used at the beginning of
   <data name="Convert_to_record" xml:space="preserve">
     <value>Convert to record</value>
   </data>
+  <data name="Introduce_parameter" xml:space="preserve">
+    <value>Introduce parameter</value>
+  </data>
+  <data name="Introduce_parameter_for_0" xml:space="preserve">
+    <value>Introduce parameter for '{0}'</value>
+  </data>
+  <data name="Introduce_parameter_for_all_occurrences_of_0" xml:space="preserve">
```

Comment:
so it looks like we offer three separate fixes, each with a fix-all-occurrences varaint.  I understand Introduce_new_parameter_overload_for_0 and Introduce_parameter_and_extract_method_for_0, but what is the vanilla Introduce_parameter_for_0 case?

Reply:
n/m.  i know what case this is.